src/AutoRest.CSharp.V3/JsonRpc/StandaloneAutoRestInterface.cs

```diff
@@ -59,6 +59,10 @@ public Task<string[]> ListInputs(string? artifactType = null)
 
         public async Task WriteFile(string filename, string content, string artifactType, RawSourceMap? sourceMap = null)
         {
+            if (string.IsNullOrEmpty(content))
+            {
+                return;
+            }
```

Comment:
This guard clause should be in the other override (below) too.

Reply:
We don't use that one, removed.